refl1d/probe.py

```diff
@@ -1116,7 +1119,8 @@ def load4(filename, keysep=":", sep=None, comment="#", name=None,
     else:
         order = [0, 1, 2, 3]
     def _as_Qprobe(data):
-        Q, R, dR, dQ = (data[1][k] for k in order)
+        lower_lim,upper_lim=data_range
+        Q, R, dR, dQ = [data[1][k][lower_lim:upper_lim] for k in order]
```

Comment:
Lint will complain about spacing: `lower_lim, upper_lim = data_range`.  

Can allow selection of every nth data point using the following:
```python
index = slice(*data_range)
Q, R, dR, dQ = [data[1][k][index] for k in order]
```